platform-api/che-core-api-builder/src/main/java/org/eclipse/che/api/builder/BuilderSelectionStrategy.java

```diff
@@ -22,4 +23,8 @@
  */
 public interface BuilderSelectionStrategy {
     RemoteBuilder select(List<RemoteBuilder> slaveBuilders);
+
+    default RemoteBuilder select(List<RemoteBuilder> slaveBuilders, BaseBuilderRequest request){
+        return select(slaveBuilders);
+    }
```

Comment:
I suggest providing a default implementation for the shorter version, this way we require minimal implementation code for implementations that rely on either version of the method.
Currently there are only two builder selection strategies, changing them is one-liner ..